### ISSUE TITLE
Soil data not matching desktop app occasionally

### ISSUE THREAD

**Issue by barrc** [Eastern-Research-Group/SWC#7]
The cutoffs are different for displaying the slope soils data.  On the desktop the cutoffs are as follows:
Flat: <=2%
Moderately flat: 2-7%
Moderately steep: 7-15%
Steep: >=15%

```
public static string GetSlopeGroup(double slope)
{
if (slope <= 2) return "A";
if (slope <= 7) return "B";
if (slope <= 15) return "C";
return "D";
}
```

Desktop slope and key:
![image](https://user-images.githubusercontent.com/13906772/49671340-65474d00-fa1c-11e8-9889-8e3762f6c559.png)
![image](https://user-images.githubusercontent.com/13906772/49671346-67a9a700-fa1c-11e8-81b9-a2128ce3cb89.png)

Web slope and key:
![image](https://user-images.githubusercontent.com/13906772/49671353-6bd5c480-fa1c-11e8-9276-43816121baeb.png)
![image](https://user-images.githubusercontent.com/13906772/49671358-6e381e80-fa1c-11e8-91b2-ea4f56c35f41.png)

**Comment by barrc** [comment]
@JasonBernagros commented:
We want the cutoffs used in the SWC desktop to be used in the SWC web app

@michelleannesimon commented:
Yes, we want the soils to be the same - Do we want to make the color schemes the same for both mobile app and desktop app?

@JasonBernagros -- would you like to standardize the color schemes?

**Comment by JasonBernagros** [comment]
I don’t think we need to have the color schemes match between the desktop and web app. It’s most important that the cutoffs are the same for both applications.

​​
Jason T. Bernagros
Landscape Architect
LEED AP BD + C
US EPA Office of Research and Development
National Risk Management Research Laboratory
Water Systems Division, Water Resources Recovery Branch
O: Reagan Building, Desk 71263
P: 202-566-1671
E: bernagros.jason@epa.gov<mailto:bernagros.jason@epa.gov>
Mail: 1200 Pennsylvania Ave. NW, MC 8623R, Washington, DC  20460
Delivery: 1300 Pennsylvania., NW (Reagan Building, Desk 71263), Washington, DC  20004
W: (Green Infrastructure) http://www2.epa.gov/water-research/green-infrastructure-research
      (Water Innovation & Technology) https://www.epa.gov/water-innovation-tech
